App/Services/AuthenticateService.php

```diff
@@ -2,12 +2,21 @@
 
 namespace App\Services;
 
-use App\Models\UsersModel;
-use League\Route\Http\Exception;
+use Cartalyst\Sentinel\Native\SentinelBootstrapper;
 use Zend\Diactoros\ServerRequest;
+use Cartalyst\Sentinel\Native\Facades\Sentinel;
 
 class AuthenticateService
 {
+    private $sentinel;
+
+    public function __construct()
+    {
+        $config = include BASEPATH . '/App/Config/sentinelConfig.php';
+        $sentinelBootstrapper = new SentinelBootstrapper($config);
+        $facade = new Sentinel($sentinelBootstrapper);
+        $this->sentinel = $facade->getSentinel();
```

Comment:
We are going to leverage dependency injection here. It is not the responsibility of the Auth service to bootstrap sentinel.